ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -120,8 +146,10 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 				<Text style={this.styles().normalText}>{_('Created: %s', time.formatMsToLocal(mk.created_time))}</Text>
 				<View style={{ flexDirection: 'row', alignItems: 'center' }}>
 					<Text style={{ flex: 0, fontSize: theme.fontSize, marginRight: 10, color: theme.color }}>{_('Password:')}</Text>
-					<TextInput selectionColor={theme.textSelectionColor} secureTextEntry={true} value={password} onChangeText={text => onPasswordChange(text)} style={inputStyle}></TextInput>
+					<TextInput selectionColor={theme.textSelectionColor} secureTextEntry={secureTextEntry} value={password} onChangeText={text => onPasswordChange(text)} style={inputStyle}></TextInput>
 					<Text style={{ fontSize: theme.fontSize, marginRight: 10, color: theme.color }}>{passwordOk}</Text>
+					<Button title={'👁️'} onPress={()=>this.handlePasswordShow(mk.id)}  ></Button>
+					<Text>  </Text>
```

Comment:
No empty `Text` please. CSS is there for margins.

Please don't use an emoji for the button as that will render differently, and sometimes weirdly, on different devices. Use an icon.

Please make it look like the button is inside the text input and remove the blue background (just use a transparent background).